mula/scheduler/queues/boefje.py

```diff
@@ -21,4 +21,7 @@ def create_hash(self, p_item: models.PrioritizedItem) -> str:
         input_ooi = dict_utils.deep_get(p_item.model_dump(), ["data", "input_ooi"])
         organization = dict_utils.deep_get(p_item.model_dump(), ["data", "organization"])
 
-        return mmh3.hash_bytes(f"{input_ooi}-{boefje_id}-{organization}").hex()
+        if input_ooi:
```

Comment:
This change will allow for the execution of boefjes that don't have a related OOI